### ISSUE TITLE
Cull modules and providers that are unused or dead

### ISSUE THREAD

**Issue by Yantrio** [opentofu/registry#63]
This could be forks of projects that have no extra commits on top of their upstream, or just old modules that we see no usage from.

**Comment by Yantrio** [comment]
We may also want to look at those that are forks with 0 or 1 stars.

**Comment by cam72cam** [comment]
I think @roni-frantchi had a good workflow that we instead look for any tags that are in the fork that aren't in the main repo.  That seems like the safest check to me as we know a lot of forks that are used don't have significant *starrage*

**Comment by roni-frantchi** [comment]
Yes, I've used GitHub GraphQL to hit the search API fetching not only the repo URLs but also their creation date and their latest 3 tags' count+creation date.  

Forks include their upstream tags - and forks with no new tags are can never never have changes from upstream that can actually  be consumed

**Comment by cam72cam** [comment]
In practice, the overhead of running the registry (even with the dupes) is low enough that we are not considering adding this requirement.
